app/src/main/java/org/p2p/wallet/jupiter/interactor/SwapTokensInteractor.kt

```diff
@@ -30,15 +30,12 @@ class SwapTokensInteractor(
     }
 
     suspend fun getAllTokens(): List<SwapTokenModel> {
-        val userTokens = tokenServiceCoordinator.getUserTokens()
+        val userTokens = tokenServiceCoordinator.getUserTokens().map(SwapTokenModel::UserToken)
```

Comment:
I suggest using set instead of list for filtering

```suggestion
        val userTokens = tokenServiceCoordinator.getUserTokens().map(SwapTokenModel::UserToken)
        val userTokenMintAddresses = userTokens.map { it.mintAddress }.toSet()
        
        val jupiterTokens = swapTokensRepository.getTokens()
            .map(SwapTokenModel::JupiterToken)
            .filter { it.mintAddress !in userTokenMintAddresses }
```